### ISSUE TITLE
Problem with pre-trained model

### ISSUE THREAD

**Issue by ThorsteinnJonsson** [qianguih/voxelnet#18]
There seems to be an issue with the pre-trained model included (save_model/pre_trained_car). 
When I run test.py, the visulizations seem to give highly inaccurate results. Furthermore, the ground truth boxes also seem to be flipped along the y-axis in the top-down lidar  visualizations.
Is there something wrong with the pre-trained model or coordinate conversions?
Below are two example results.

![000001_bv](https://user-images.githubusercontent.com/33022898/39766450-4e8499d2-531f-11e8-86a6-64c85a257da6.jpg)
![000001_front](https://user-images.githubusercontent.com/33022898/39766472-5753f76a-531f-11e8-9a9f-4278c923b0cf.jpg)

![000002_bv](https://user-images.githubusercontent.com/33022898/39766483-5c9b6780-531f-11e8-8aff-77795f975a7a.jpg)
![000002_front](https://user-images.githubusercontent.com/33022898/39766487-5ea69ee6-531f-11e8-914c-3b59f4c82fd3.jpg)

**Comment by ArpitaSTugave** [comment]
I got the correct predictions. My guess would be your calibration values are wrong. Please update according to calibrated (synced) kitti data.

**Comment by sakshi404** [comment]
Got the error while running pre-trained model:
FailedPreconditionError : Attempting to use uninitialized value MiddleAndRPN_/conv20/bias
was able to run it properly in next iteration, but again getting the same error.
con12/beta or conv2/bias uninitialized .

What should be the value of available_gpu parameter if we are running on CPU? Do we have to change other things if model is run on cpu.

**Comment by sainisanjay** [comment]
@sakshi404, i am also getting same error, did you solve the above error????

**Comment by sainisanjay** [comment]
@ThorsteinnJonsson @qianguih  may i know how to visualise the results. Do i have to pass any flag?? I am running the test script:  `python3 test.py -n pre_trained_car`
using this it save the result in `./predictions/data` in .txt files.

**Comment by angiend** [comment]
@ThorsteinnJonsson @qianguih @sainisanjay
thanks,Would you share the method to visualize the results?appreciate it

**Comment by angiend** [comment]
> There seems to be an issue with the pre-trained model included (save_model/pre_trained_car).
> When I run test.py, the visulizations seem to give highly inaccurate results. Furthermore, the ground truth boxes also seem to be flipped along the y-axis in the top-down lidar visualizations.
> Is there something wrong with the pre-trained model or coordinate conversions?
> Below are two example results.
> 
> ![000001_bv](https://user-images.githubusercontent.com/33022898/39766450-4e8499d2-531f-11e8-86a6-64c85a257da6.jpg)
> ![000001_front](https://user-images.githubusercontent.com/33022898/39766472-5753f76a-531f-11e8-9a9f-4278c923b0cf.jpg)
> 
> ![000002_bv](https://user-images.githubusercontent.com/33022898/39766483-5c9b6780-531f-11e8-8aff-77795f975a7a.jpg)
> ![000002_front](https://user-images.githubusercontent.com/33022898/39766487-5ea69ee6-531f-11e8-914c-3b59f4c82fd3.jpg)
how to visiualise the results?thanks

**Comment by gkadusumilli** [comment]
> @ThorsteinnJonsson @qianguih may i know how to visualise the results. Do i have to pass any flag?? I am running the test script: `python3 test.py -n pre_trained_car`
> using this it save the result in `./predictions/data` in .txt files.

for visualization run python3 test.py -n pre_trained_car --vis==True

**Comment by SamudhbhavPrabhu** [comment]
@ThorsteinnJonsson Even I am getting the same results. Were you able to solve this issue?
@ArpitaSTugave Could you please post how your results look like for the first 2 images (for the same ones posted above by @angiend  and @ThorsteinnJonsson )?

**Comment by shreyas1998** [comment]
@sakshi404 , I am experiencing the same issue. What do we need to change to run the model on cpu?
